examples/sampling-slice-adaptive-hyperrectangles-mcmc.ipynb

```diff
@@ -0,0 +1,948 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Inference: Slice Sampling with Adaptive Hyperrectangles\n",
+    "This example shows you how to perform Bayesian inference on multiple toy problems, using\n",
```

Comment:
Change to, "This example shows how to perform Bayesian inference on multiple toy problems..."